### PR TITLE
JavaScript implementations

### PR DESCRIPTION
**Pull request by Otegaa**
This PR is for the JS implementations of my portfolio where I work on the hamburger menu pop-up.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has no Future Features section on your README and have templated texts like below. Please remove all templated texts from your README sections. Please don't forget to add at least one future feature.
Screenshot 2023-03-06 at 9 44 57 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -15,10 +15,25 @@
...
+            src="./assets/icons/cancelHamburger.png"
+            alt=""
+            class="close-cross"
+          />
+        </div>
+        <ul class="show-links-hamburger">
+          <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
+          <li><a href="#about-info" class="menu-links">About</a></li>
+          <li><a href="#contact" class="menu-links">Contact</a></li>
+        </ul>
```
Comment (AbayomiOlaoye) [inline index.html]: - Excellent job! :clap: :clap: All the functions are well implemented. :vulcan_salute: 
- [ ] However, we do have a few disparity in your choice of styling when compared with the Figma design. There seem to be too much `padding` and `margin-top` on your navigation links. I found a `padding: 32px` and a `margin-top: 24px` on each while all you have on the Figma is just a spacing value of `24px`/`1.5rem`. See the Image below:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|
- Kindly fix this to match design, and to also improve your user interface, as well as the experience users have with accessing your page :wink:

File: index.html [hunk 2]
```diff
@@ -15,10 +15,25 @@
...
+        <div class="cross-container">
+          <img
+            src="./assets/icons/cancelHamburger.png"
+            alt=""
+            class="close-cross"
+          />
+        </div>
+        <ul class="show-links-hamburger">
+          <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
+          <li><a href="#about-info" class="menu-links">About</a></li>
```
Comment (AbayomiOlaoye) [inline index.html]: - [ ] For ease of access and smooth navigation experience to users or visitors, I think you should consider redirecting the to your `About me` section whenever they click on this link. The link is currently referencing the first page/landing page when what I wanted to navigate to was your About me section, just before the `Contact section`. Please consider removing the `id` from the first page and add it to the section where you have this view :point_down: 
image

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @Otegaa :wave: , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- DOM manipulation works well :heavy_check_mark: 
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- You can also consider:
  - [ ] Fixing your broken email icon whose image is not displaying on larger screens as shown below:
 
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by huseyinbiyik — APPROVED** [review]
Hi team,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Eslint set up well and checks pass 🟢
- Github flow followed 👌
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
- I think, there is a bit more space around your nav. I recommend you to decrease a little bit to match the Figma template 👍 
Screenshot 2023-03-06 at 11 25 46 PM
Screenshot 2023-03-06 at 11 26 44 PM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**